cinema/tests/test_movie_api.py

```diff
@@ -157,3 +158,156 @@ def test_image_url_is_shown_on_movie_session_detail(self):
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
```

Comment:
The test `test_image_url_is_shown_on_movie_session_detail` checks for 'movie_image' in the response keys. Ensure that the API actually returns this key, as it might be a custom implementation detail.